app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getTopic } = require("./controller/getTopic.controller");
+const { getArticleByiD } = require("./controller/getArticle.controller");
+
+const app = express();
+
+app.get("/api/topics", getTopic);
+app.get("/api/articles/:article_id", getArticleByiD);
+
```

Comment:
can we rename getArticlebyiD so it follows camel case rules properly (getArticleById)